### PR TITLE
Add new themed pages for 1.27 release

### PR DESCRIPTION
**Pull request by bguyza**
Theme page documentation for:
* Email verification required
* Verify registration required

### CODE REVIEW

File: site/docs/v1/tech/themes/index.adoc [hunk 1]
```diff
@@ -110,15 +110,20 @@ This page is used after a user has verified their email address by clicking the
...
 
+[field]#Email verification required# [required]#Required# [since]#Since 1.27.0#::
+This page is rendered when a user is required to verify their email address prior to being allowed to proceed with login. This occurs when [field]#Unverified behavior# is set to `Gated` in email verification settings on the Tenant.
++
+[uri]#/email/verify#
+
 [field]#Email verification# [required]#Required#::
 This page is rendered when a user clicks the URL from the verification email and the `verificationId` has expired. FusionAuth expires `verificationId` after a period of time (which is configurable). If the user has a URL from the verification email that has expired, this page will be rendered and the error will be displayed to the user.
 +
-[uri]#/email/verify#
```
[the remark this answers is not in the document]
Comment (bguyza) [reply site/docs/v1/tech/themes/index.adoc:121]: Yep just mistakenly changed the wrong one.

**Comment by mooreds** [comment]
Just added an email templates update doc task.

File: site/docs/v1/tech/themes/index.adoc [hunk 2]
```diff
@@ -661,6 +671,29 @@ The email address that was passed as a URL parameter. This is the email address
 [field]#emailSent# [type]#[Boolean]#::
 A boolean that indicates if the verification email was re-sent or not.
 
+=== Email verification required
```
Comment (mooreds) [inline site/docs/v1/tech/themes/index.adoc:674]: `Email verification re-send` should be `Email verification re-sent` (t on the end) to match what is in the UI

Comment (mooreds) [reply site/docs/v1/tech/themes/index.adoc:674]: Couldn't put it higher, sorry.

**Review by mooreds — APPROVED** [review]
One small typo in a related section, other than that looks good.

File: site/docs/v1/tech/themes/index.adoc (hunk 2 above)
Comment (bguyza) [reply site/docs/v1/tech/themes/index.adoc:674]: Thanks Dan, I fixed those typos. I'll merge in a bit.

**Pull request merged**